packages/web/components/control/page-list.tsx

```diff
@@ -14,6 +15,7 @@ export const PageList: FunctionComponent<Props> = ({
   min,
   max,
   editField = false,
+  className,
 }) => {
   const [textEditing, setIsEditing] = useState(false);
```

Comment:
I think it would be clearer to add `set` as prefix to state variable name itself.  Also, I recommend to name variable `Is~~` for boolean.